packages/xchain-bsc/__e2e__/bsc-client.e2e.ts

```diff
@@ -0,0 +1,146 @@
+import { Balance, Network, TxType } from '@xchainjs/xchain-client'
+import { ApproveParams, EstimateApproveParams, IsApprovedParams } from '@xchainjs/xchain-evm'
+import { Asset, assetAmount, assetToBase, assetToString } from '@xchainjs/xchain-util'
+
+import BscClient from '../src/client'
+import { AssetBSC, BSCChain, defaultBscParams } from '../src/const'
+
+// import { ApproveParams, EstimateApproveParams, IsApprovedParams } from '../src/types'
+
+// =====Erc-20 asset=====
+
+const assetRIP: Asset = {
+  chain: BSCChain,
+  symbol: `RIP-0x224695ba2a98e4a096a519b503336e06d9116e48`,
```

Comment:
does this token even exist on the bsc network?
have you run these e2e tests with real funds?

Reply:
No I haven't made any e2e tests I just copied this from the AVAX code. I can make e2e tests next